tests/regression/test_fdm.py

```diff
@@ -222,6 +222,7 @@ def fs(request, mesh):
 
 @pytest.mark.skipcomplex
 def test_ipdg_direct_solver(fs):
+    from pyop2.utils import as_tuple
```

Comment:
This import should be at the top of the file. Imports should only be used inside functions if required to resolve a circular import.